### ISSUE TITLE
Performance on the validation set of something-something-v1

### ISSUE THREAD

**Issue by pplntech** [zhang-can/ECO-pytorch#30]
Hi, thank you for releasing pytorch version of ECO.

Have you run ECO-pytorch on the validation set of something-something-v1 dataset?

I wonder if it have reached the performance on the paper, 46.4%, which is obtained from an ensemble of networks with {16, 20, 24, 32} number of frames.

Thank you.

**Comment by chen-ming2019** [comment]
@pplntech can you share the downloaded something -something-v1 dataset with me ?My downloaded dataset has been not using.

**Comment by anti199994** [comment]
@chen-ming2019 请问在something数据集复现，为何我的损失在４多不变了，准确率只有4%　是我的参数那里设置错了吗？还是什么问题

**Comment by chen-ming2019** [comment]
@anti1990 我没有跑ECO在something数据集上，但你这个结果应该是有问题的。你可以看看是不是模型最后的全连接层的类别数没有改？
